manager-parent/manager-core/src/main/resources/application.properties

```diff
@@ -10,19 +10,6 @@ rhose.metrics-name.operation-duration-seconds=managed_services_api_rhose_operati
 # Cors
 quarkus.http.cors=true
 
-# Swagger and openapi
-quarkus.swagger-ui.always-include=true
-quarkus.smallrye-openapi.store-schema-directory=target/generated/
-quarkus.smallrye-openapi.info-title=Red Hat Openshift SmartEvents Fleet Manager
-quarkus.smallrye-openapi.info-version=0.0.1
-quarkus.smallrye-openapi.info-description=The API exposed by the fleet manager of the SmartEvents service.
-quarkus.smallrye-openapi.info-license-name=Apache 2.0
-quarkus.smallrye-openapi.info-license-url=https://www.apache.org/licenses/LICENSE-2.0
-quarkus.smallrye-openapi.info-contact-email=openbridge-dev@redhat.com
-quarkus.smallrye-openapi.info-contact-name=Development Team
-quarkus.smallrye-openapi.servers=https://api.openshift.com,https://api.stage.openshift.com,http://localhost:8000,/
-mp.openapi.extensions.smallrye.operationIdStrategy=CLASS_METHOD
```

Comment:
I think I'm missing something. Do we have this config somewhere else?

Reply:
No.. I meant to add it to the `manager-v1` properties.. but, based on the failing build, I forgot.

Reply:
I guess, it'd be safer keeping it on `manager-core`.... so I'll add it back.

Reply:
I also added an _explicit_ dependency for `io.quarkus:quarkus-smallrye-openapi`.